### PR TITLE
VACMS-15702 Fix view mode notifications

### PR DESCRIPTION
**Pull request by swirtSJW**
## Description

Relates to #15702

image

## QA steps

What needs to be checked to prove this works?
What needs to be checked to prove it didn't break any related things?
What variations of circumstances (users, actions, values) need to be checked?

As an admin
1. Flush the cache. (these messages often only appear on the first load of a page, why they appear after save).
2. Then visit the follow pages and validate that you see no notification about field office hours.
   - [x] Validate [VBA](https://pr15863-eguzdu2tctxueapcxxmcktk80nnprx1v.ci.cms.va.gov/vba-facilities/locations/va-regional-benefit-satellite-office-at-fort-smith-va-clinic)
   - [x] Validate [NCA](https://pr15863-eguzdu2tctxueapcxxmcktk80nnprx1v.ci.cms.va.gov/node/4481)
   - [x] Validate [VAMC](https://pr15863-eguzdu2tctxueapcxxmcktk80nnprx1v.ci.cms.va.gov/node/1941)
   - [x] Validate [Vet Center](https://pr15863-eguzdu2tctxueapcxxmcktk80nnprx1v.ci.cms.va.gov/node/3776)
   - [x] Validate [VC Mobile](https://pr15863-eguzdu2tctxueapcxxmcktk80nnprx1v.ci.cms.va.gov/henderson-vet-center/henderson-mobile-vet-center)
   - [x] Validate [VC Outstation](https://pr15863-eguzdu2tctxueapcxxmcktk80nnprx1v.ci.cms.va.gov/la-crosse-vet-center/wausau-vet-center-outstation)
   - [x] Validate [Billing and Insurance](https://pr15863-eguzdu2tctxueapcxxmcktk80nnprx1v.ci.cms.va.gov/eastern-kansas-health-care/billing-and-insurance)
   - [x] Validate [VC CAP](https://pr15863-eguzdu2tctxueapcxxmcktk80nnprx1v.ci.cms.va.gov/dubois-vet-center/community-access-point/dubois-vet-center-penn-state-dubois)

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.
- [ ] If there are field changes, front end output has been thoroughly checked.

### Select Team for PR review

- [ ] `CMS Team`
- [ ] `Public websites`
- [ ] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping the UX writer so they are ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping the UX writer to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Comment by swirtSJW** [comment]
I'll add qa steps in the mornin.

File: config/sync/core.entity_view_display.node.vba_facility.ief_table.yml [hunk 1]
```diff
@@ -104,29 +104,30 @@ mode: ief_table
 content:
   field_address:
     type: address_default
-    label: above
```
Comment (swirtSJW) [inline config/sync/core.entity_view_display.node.vba_facility.ief_table.yml:107]: These changes seem odd.  These should not have changed.

File: config/sync/core.entity_view_display.node.vba_facility.ief_table.yml [hunk 2]
```diff
@@ -104,29 +104,30 @@ mode: ief_table
...
+    label: hidden
```
Comment (swirtSJW) [inline config/sync/core.entity_view_display.node.vba_facility.ief_table.yml:107]:
```suggestion
    label: above
```

File: config/sync/core.entity_view_display.node.vba_facility.ief_table.yml [hunk 3]
```diff
@@ -104,29 +104,30 @@ mode: ief_table
...
     settings:
       link_to_entity: false
     third_party_settings: {  }
     weight: 3
     region: content
   field_office_hours:
     type: office_hours
-    label: above
+    label: hidden
```
Comment (swirtSJW) [inline config/sync/core.entity_view_display.node.vba_facility.ief_table.yml:122]:
```suggestion
    label: above
```

File: config/sync/core.entity_view_display.node.vba_facility.ief_table.yml [hunk 4]
```diff
@@ -150,21 +152,21 @@ content:
     region: content
   field_operating_status_facility:
     type: list_default
-    label: above
+    label: hidden
```
Comment (swirtSJW) [inline config/sync/core.entity_view_display.node.vba_facility.ief_table.yml:155]:
```suggestion
    label: above
```

File: config/sync/core.entity_view_display.node.vba_facility.ief_table.yml [hunk 5]
```diff
@@ -150,21 +152,21 @@ content:
...
     settings: {  }
     third_party_settings: {  }
     weight: 2
     region: content
   field_operating_status_more_info:
     type: basic_string
-    label: above
+    label: hidden
```
Comment (swirtSJW) [inline config/sync/core.entity_view_display.node.vba_facility.ief_table.yml:162]:
```suggestion
    label: above
```

File: config/sync/core.entity_view_display.node.vba_facility.ief_table.yml [hunk 6]
```diff
@@ -150,21 +152,21 @@ content:
...
     settings: {  }
     third_party_settings: {  }
     weight: 3
     region: content
   field_phone_number:
     type: telephone_link
-    label: above
+    label: hidden
```
Comment (swirtSJW) [inline config/sync/core.entity_view_display.node.vba_facility.ief_table.yml:169]:
```suggestion
    label: above
```

File: config/sync/core.entity_view_display.node.vba_facility.ief_table.yml [hunk 7]
```diff
@@ -201,6 +203,8 @@ hidden:
   flag_changed_title: true
   flag_new: true
   flag_removed_from_source: true
+  label: true
```
Comment (swirtSJW) [inline config/sync/core.entity_view_display.node.vba_facility.ief_table.yml:206]:
```suggestion
```

File: config/sync/core.entity_view_display.node.vba_facility.ief_table.yml [hunk 8]
```diff
@@ -201,6 +203,8 @@ hidden:
...
   langcode: true
   links: true
   search_api_excerpt: true
+  status: true
```
Comment (swirtSJW) [inline config/sync/core.entity_view_display.node.vba_facility.ief_table.yml:210]:
```suggestion
```

**Review by omahane — APPROVED** [review]
Everything checks out. Editors will be happy to not have an inscrutable message at the top of their edit screen.

**Pull request merged**